Sync Community PR #2041 ([v2.13.1] Rancher IPv6 Support) (#619)
rancher/rancher-product-docs: versions/v2.13/modules/en/pages/cluster-deployment/infra-providers/aws/machine-configuration.adoc

=== Add Tag

Add metadata using https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html[tags] to categorize resources.

=== IPv6 Address Count

Specify how many IPv6 addresses to assign to the instance’s network interface.

=== IPv6 Address Only

> Enable this option if the instance should use IPv6 exclusively. IPv6-only VPCs or subnets require this. When enabled, the instance will have IPv6 as its sole address, and the IPv6 Address Count must be greater than zero.

Reviewer: ```suggestion
Enable this option if the instance should use IPv6 exclusively. IPv6-only VPCs or subnets require this. When enabled, the instance has IPv6 as its sole address, and the IPv6 Address Count must be greater than zero.
```
